central/metadata/service/service_impl.go

```diff
@@ -186,3 +187,12 @@ func (s *serviceImpl) GetDatabaseBackupStatus(ctx context.Context, _ *v1.Empty)
 		BackupInfo: sysInfo.GetBackupInfo(),
 	}, nil
 }
+
+// GetCentralCapabilities returns central services capabilities.
+func (s *serviceImpl) GetCentralCapabilities(ctx context.Context, _ *v1.Empty) (*v1.CentralServicesCapabilities, error) {
+	if authn.IdentityFromContextOrNil(ctx) == nil {
```

Comment:
I think it's important to consider when this endpoint will be called in UI code - we have some endpoints(for example, `/v1/metadata` which are called without the user logging in).

One corner case that comes to mind is if we decide to render the login page differently depending on central capabilities

Reply:
As discussed in call, won't implement unauthenticated fields for now without the actual use.